extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -76,11 +76,11 @@ export class AmpImageViewer extends AMP.BaseElement {
     /** @private {number} */
     this.sourceHeight_ = 0;
 
-    /** @private {!../../../src/layout-rect.LayoutRectDef} */
-    this.elementBox_ = layoutRectLtwh(0, 0, 0, 0);
+    /** @private {?../../../src/layout-rect.LayoutRectDef} */
```

Comment:
I'm now a bigger fan of being explicit about this value not being initialized as opposed to giving it a valid but incorrect (0,0,0,0) value.